lib/Ray/include/Drawables/ADrawable3D.hpp

```diff
@@ -0,0 +1,41 @@
+/*
+** EPITECH PROJECT, 2021
+** Bomberman
+** File description:
+** ADrawable
+*/
+
+#ifndef ADRAWABLE3D_HPP_
+#define ADRAWABLE3D_HPP_
+
+#include <raylib.h>
+#include <Vector.hpp>
+
+namespace RAY::Drawable3D {
+    class ADrawable3D
+	{
+        public:
+            //! @param Color Color of the drawable  
+            ADrawable3D(Color color);
+
+            //! @brief A default copy constructor
+            ADrawable3D(const ADrawable3D &) = default;
+
+            //! @brief A default destructor
+            virtual ~ADrawable3D() = 0;
```

Comment:
Your destructor should be defaulted, not disabled. Disabled destructors are not really useful. (Maybe do a search for other disabled destructors that we may have not seen)